src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -235,7 +237,8 @@ public void mainDrive(Vector spdVec, double xRot) {
    * @return the angle of the robot gyro
    */
   public Measure<Angle> getGyroAngle() {
-    return Units.Degrees.of(m_gyro.getAngle());
+    m_heading = m_heading.mut_replace(m_gyro.getAngle(), Units.Degrees);
```

Comment:
Why not just `return m_heading.mut_replace(m_gyro.getAngle(), Units.Degrees);`?

Reply:
I don't actually know, though this way it is obvious that the value of the measure is being changed

Reply:
> I don't actually know, though this way it is obvious that the value of the measure is being changed

I think it's already evident since you're using `mut_replace` I don't like the self-overwriting.